flink-scala/src/main/scala/org/apache/flink/api/scala/codegen/TypeInformationGen.scala

```diff
@@ -320,14 +315,6 @@ private[flink] trait TypeInformationGen[C <: Context] {
     }
   }
 
-  def mkWritableTypeInfo[T <: Writable : c.WeakTypeTag](
```

Comment:
What exactly does this removal mean for supporting `Writable`? Does hadoop-compat take care of that?

Reply:
Unfortunately, this means that users now have to manually specify a `TypeInformation` that they can get from `TypeExtractor.createHadoopWritableTypeInfo(MyWritable.class)`.

I'm not sure how often people are using Hadoop Writables in their Scala code but this is definitely something that will break.

Reply:
Scratch that, this actually still works and I added a test for that in the Hadoop compat package.